lib/runtime-core/src/table/anyfunc.rs

```diff
@@ -128,4 +128,12 @@ impl AnyfuncTable {
             Err(())
         }
     }
+
+    pub fn get(&mut self, index: u32) -> Result<vm::Anyfunc, ()> {
+        if let Some(slot) = self.backing.get(index as usize) {
+            Ok(*slot)
+        } else {
+            Err(())
```

Comment:
I think it might be a good idea to return an Option rather than a Result (since there is only one kind of error: if the function doesn't exist).